### DISCUSSION TITLE
How do I disable .btn-square on larger screens?

### DISCUSSION THREAD

**Discussion by gwesterman** [saadeghi/daisyui#2225] [Q&A] [answered]
I have a button that should be square and only display an icon on small screens and have its default styling and show both an icon and text on larger screens.

```html
<!-- mock -->
<button class="btn btn-sm btn-square">
  <i>...</i>
  <span class="hidden lg:inline">Text</span>
</button>
```

I would need to add a class such as `lg:btn-default` to revert this choice on larger screens.

```html
<!-- mock -->
<button class="btn btn-sm btn-square lg:btn-default">
  <i>...</i>
  <span class="hidden lg:inline">Text</span>
</button>
```

What am I missing here?

**Comment by saadeghi** [comment, marked as answer]
Tailwind also has a `max-lg:` which works opposite of `lg:`  

So you just need:
```html
<!-- mock -->
<button class="btn btn-sm max-lg:btn-square">
  <i>...</i>
  <span class="hidden lg:inline">Text</span>
</button>
```

Let me know if you have any questions
